pybossa/data_access.py

```diff
@@ -247,5 +247,7 @@ def valid_user_type_based_data_access(user_type, access_levels):
 @when_data_access()
 def set_default_amp_store(project):
     annotation_config = project['info'].get('annotation_config', {})
-    annotation_config['amp_store'] = True
+    input_data = project['info'].get('input_data') or ''
+    output_data = project['info'].get('output_data') or ''
```

Comment:
fyi, input and output data are part of project.info.data_classification. computed data access from input, output data classification is updated under project.info.data_access can be utilized here

Reply:
I see, thanks for pointing out. 
When user create project with API, do they fill data_classification.input_data/output_data fields or data_access field?